### PR TITLE
Rename `getconf.py` to `pbench-config`

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: server/bin/pbench-config
Comment (webbnh) [inline server/bin/pbench-config:1]: Should this be named `pbench-server_config` (in light of #3343)?

**Review by portante — COMMENTED** [review]
Ready for review.

File: server/rpm/pbench-server.spec.j2 [hunk 1]
```diff
@@ -125,14 +125,14 @@ fi
...
 %defattr(755, pbench, pbench, 755)
-/%{installdir}/bin/getconf.py
 /%{installdir}/bin/pbench-audit-server
 /%{installdir}/bin/pbench-audit-server.sh
 /%{installdir}/bin/pbench-base.py
 /%{installdir}/bin/pbench-check-tb-age
 /%{installdir}/bin/pbench-check-tb-age.py
 /%{installdir}/bin/pbench-clean-up-dangling-results-links
 /%{installdir}/bin/pbench-clean-up-dangling-results-links.sh
+/%{installdir}/bin/pbench-server-config
```
Comment (webbnh) [inline server/rpm/pbench-server.spec.j2:135]: Why did you insert this at line 135 instead of, say, at line 141?

File: server/test/run [hunk 2]
```diff
@@ -4,9 +4,9 @@
 # platforms (well, mostly matches on all platforms).
 umask 0002
 
-# Don't allow an external _PBENCH_SERVER_CONFIG environment variable to affect the operation
-# of the unit tests since the unit tests themselves use getconf.py which looks
-# for that envinronment variable as input.
+# Don't allow an external _PBENCH_SERVER_CONFIG environment variable to affect
+# the operation of the unit tests since the unit tests themselves use pbench-
+# config which looks for that envinronment variable as input.
```
Comment (webbnh) [inline server/test/run:9]: s/pbench-\nconfig/pbench-server-config/ ?

**Review by webbnh — APPROVED** [review]
Looks good except for a couple of nits.

**Review by ndokos — APPROVED** [review]

File: server/test/run (hunk 2 above)
Comment (portante) [reply server/test/run:9]: I'll fix in the next commit.

File: server/rpm/pbench-server.spec.j2 (hunk 1 above)
Comment (portante) [reply server/rpm/pbench-server.spec.j2:135]: I'll fix in another commit shortly.

**Pull request merged**